tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py310-django{32,42}
+    py310-django42
     py311-django42
```

Comment:
```suggestion
    py3{10,11)-django42
```

Reply:
placing the brackets there returned errors, so I changed it to `py{310,311}-django42`